### PR TITLE
PUBDEV-4343: Documentation: Update the stacking online tutorial

### PR DESCRIPTION
**Pull request by hannah-tillman**
Switched the example over to the Stacked Ensemble example from the user guide. Since I'm currently unable to build R documentation locally, I tried to follow the guidelines of the previous example as closely as I could. Please let me know if you want me to add anything back in or change what's in there.

### CODE REVIEW

File: tutorials/ensembles-stacking/ensembles-stacking.R [hunk 1]
```diff
@@ -111,166 +111,119 @@ h2o.removeAll() # Clean slate - just in case the cluster was already running
 #First, import a sample binary outcome train and test set into the H2O cluster.
 train <- h2o.importFile(path = normalizePath("../data/higgs_10k.csv"))
```
Comment (angela0xdata) [inline tutorials/ensembles-stacking/ensembles-stacking.R]: use real paths for train and test datasets
for example:
train <- h2o.importFile("https://s3.amazonaws.com/h2o-public-test-data/smalldata/testng/higgs_train_5k.csv")
test <-  h2o.importFile("https://s3.amazonaws.com/h2o-public-test-data/smalldata/testng/higgs_test_5k.csv")

File: tutorials/ensembles-stacking/ensembles-stacking.R [hunk 2]
```diff
@@ -111,166 +111,119 @@ h2o.removeAll() # Clean slate - just in case the cluster was already running
...
+                  learn_rate = 0.2,
+                  nfolds = nfolds,
+                  fold_assignment = "Modulo",
+                  keep_cross_validation_predictions = TRUE,
+                  seed = 1)
+#
+#Train & Cross-validate a RF
+my_rf <- h2o.randomForest(x = x,
+                          y = y,
+                          training_frame = training_frame,
```
Comment (angela0xdata) [inline tutorials/ensembles-stacking/ensembles-stacking.R]: training_frame=train,

File: tutorials/ensembles-stacking/ensembles-stacking.R [hunk 3]
```diff
@@ -111,166 +111,119 @@ h2o.removeAll() # Clean slate - just in case the cluster was already running
...
+#
+#### Evaluate ensemble performance on a test set
+perf <- h2o.performance(ensemble, newdata = test)
+#
+#Compare to base learner performance on the test set
+perf_gbm_test <- h2o.performance(my_gbm, newdata = test)
+perf_rf_test <- h2o.performance(my_rf, newdata = test)
+baselearner_best_auc_test <- max(h2o.auc(perf_gbm_test), h2o.auc(perf_rf_test))
+ensemble_auc_test <- h2o.auc(perf)
+print(sprintf("BEst Base-learner Test AUC: %s", baselearner_best_auc_test))
```
Comment (angela0xdata) [inline tutorials/ensembles-stacking/ensembles-stacking.R]: Typo: "Best..."

**Comment by ledell** [comment]
@angela0xdata Can you remind me where the text for [the stacking tutorial page](http://docs.h2o.ai/h2o-tutorials/latest-stable/tutorials/ensembles-stacking/index.html) is pulled from?  It looks like the README.md?  
@hannah-tillman It looks like the code and some text in the README.md file is what needs to be changed, not the R script. 

If this is going to be referenced widely, then it might make sense to just remove all the old h2oEnsmeble stuff from this folder -- we should find a place for it, maybe in the `./h2o-3/h2o-r/ensemble` folder in the h2o-3 repo for now?  (I should really move that stuff out of there and into it's own repo since it's very old at this point -- there's a [JIRA](https://0xdata.atlassian.net/browse/PUBDEV-4441) for this).

**Comment by angela0xdata** [comment]
@ledell oh, you're right! The file is here: https://github.com/h2oai/h2o-tutorials/blob/master/tutorials/ensembles-stacking/README.md

**Comment by hannah-tillman** [comment]
> It looks like you deleted a lot from the readme. I think (unless I'm wrong), the only real changes that needed to be done are:
> 
> * to make this work with ensemble <- h2o.stackedEnsemble instead of fit <- h2o.ensemble
> * removed text about this being implemented as a separate package
> * change the path for loading the test and train datasets to point to the smalldata folder instead of erin's folder
> 
> See the Stacked Ensemble chapter in the user guide for more info.

From what I understood from messaging @ledell was that I was to remove mention of the old package (minus some of the background information), rewrite to not talk about the wrapper functions and stuff specific to the old implementation (which was quite a bit of the old example), and switch the code example over to the one in the User Guide. Switching the example required a lot of rewriting at the bottom (especially with the explanations). I did try to add back in some of the explanations from the previous example. If there's more I should add back in, please let me know!

**Comment by angela0xdata** [comment]
@hannah-tillman ahh, I guess I either wasn't aware or I have just forgotten about that. I'll let @ledell weigh in 😬

**Comment by ledell** [comment]
@hannah-tillman Can you update the [Stacked Ensemble user guide code](http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/stacked-ensembles.html) to use the code snippet above?  There's a few edits... Thank you!

**Comment by ledell** [comment]
One thing I [changed](https://github.com/h2oai/h2o-tutorials/pull/125/commits/66f1474356b875b60c3576805afe734e80e54a31) (which you might want to change again) is the path for the two datasets.  I noticed you had another location for the data -- The results are for the 10k training set and the one you had is for the 5k training set.  If you can locate a 10k version of this data in that S3 bucket you can change it back (or add the 10k training set to the S3 bucket if you want).

**Comment by angela0xdata** [comment]
> I noticed you had another location for the data -- The results are for the 10k training set and the one you had is for the 5k training set. 
@ledell oh, I see. The 10k one isn't in smalldata or in bigdata/laptop (that I can find)

**Pull request merged**